core/trino-main/src/test/java/io/trino/server/ui/TestWebUi.java

```diff
@@ -722,20 +750,22 @@ public void testCustomPrincipalField()
         }
     }
 
-    private void assertAuth2Authentication(HttpServerInfo httpServerInfo, String accessToken)
+    private void assertAuth2Authentication(TestingTrinoServer server, String accessToken, boolean refreshTokensEnabled)
             throws Exception
     {
+        ZonedDateTime authTokenExpiration = ZonedDateTime.now().plusSeconds(30);
         String state = newJwtBuilder()
                 .signWith(hmacShaKeyFor(Hashing.sha256().hashString(STATE_KEY, UTF_8).asBytes()))
                 .setAudience("trino_oauth_ui")
-                .setExpiration(Date.from(ZonedDateTime.now().plusMinutes(10).toInstant()))
+                .setExpiration(Date.from(authTokenExpiration.toInstant()))
```

Comment:
This change is probably not needed, state is only a part of the authorization challenge https://www.rfc-editor.org/rfc/rfc6749#section-4.1.1 so its expiration shouldn't matter.

Reply:
Right, I mixed them up, reverted